src/components/PageWizard/PageWizard.test.jsx

```diff
@@ -125,4 +126,38 @@ describe('PageWizard', () => {
     const wrapper = shallow(<PageWizard currentStepId="step1">{content[0]}</PageWizard>);
     expect(wrapper.find(ProgressIndicator)).toHaveLength(0);
   });
+
+  it('i18n string tests', () => {
+    const i18nTest = {
+      back: 'back',
+      next: 'next',
+      cancel: 'cancel',
+      submit: 'submit',
+      close: 'close',
+    };
+
+    const i18nDefault = defaultProps.i18n;
+    render(
+      <PageWizard i18n={i18nTest} currentStepId="step1" error="error">
+        {content}
+      </PageWizard>
+    );
+
+    expect(screen.getByText(i18nTest.next)).toBeInTheDocument();
+    expect(screen.getByText(i18nTest.cancel)).toBeInTheDocument();
+    expect(screen.getByLabelText(i18nTest.close)).toBeInTheDocument();
+    expect(screen.queryByText(i18nDefault.next)).not.toBeInTheDocument();
+    expect(screen.queryByText(i18nDefault.cancel)).not.toBeInTheDocument();
+    expect(screen.queryByLabelText(i18nDefault.close)).not.toBeInTheDocument();
+
+    render(
+      <PageWizard currentStepId="step3" i18n={i18nTest}>
+        {content}
+      </PageWizard>
+    );
```

Comment:
I think this is going to render a second instance into the DOM instead of replacing the old one. You might want to either use `rerender` or change the currentStep by clicking/interacting with the component.